src/main/java/com/example/java/maven/gameBlackJack/Game.java

```diff
@@ -0,0 +1,96 @@
+package com.example.java.maven.gameBlackJack;
+
+public class Game {
+    Player player = new Player(UserInputProvider.getPlayersName());
+    Croupier croupier = new Croupier();
+    private final Deck deck;
+    private final static String INSTRUCTIONS =
+            ('\n' + "INSTRUCTIONS:" + '\n' +
+                    "To take card (hit)    - insert H" + '\n' +
+                    "To do nothing (stand) - insert S"
+            );
+    private final static String INVALID_INPUT = ("Invalid input! Please stick to instructions!");
+    private final String playerLost = ("No luck this time! You lost, " + player.name + "!");
+    private final String playerWon = ("You won, " + player.name + ", you lucky guy!");
+    private final static String CROUPIER_TURN = ("Croupier's turn now.");
```

Comment:
when you create method: `private String playerLostMessage(Player player)` lot of messages will be not necessary

Reply:
Done